iset.mm

```diff
@@ -60031,6 +60031,23 @@ ordinals implies excluded middle ( ~ ordsucunielexmid ).  (Contributed
     ABFZABDZABGZHZUKULBADZIUOABJUKUPUOUKUOUPUKUMUNUPKUOUPHABLUMUNUPMNOQPUKUMULU
     NUKBRZUMULSUJUQUIBTUAUQUMULBAUBUCUDUNULSUKABUEUFUGUH $.
 
+  $( A subset of a natural number belongs to its successor.  (Contributed by
```

Comment:
{0, 2} is a subset of the natural number 3 but does not belong to its successor.

The $p-statement states that a natural number is a subset of another natural number if and only if it belongs to its successor.

Reply:
Agreed, we'll need to fix https://us.metamath.org/mpeuni/ordsssuc.html too which has the same wording problem. (I'll see if I can think of something a bit more concise which is correct).

Reply:
I've pushed up a wording fix.

Reply:
Note that both $p-statements are biconditionals. I will abstain and let someone else review.

Reply:
Yeah, I noticed the biconditional and I also had a bunch more thoughts including about things like what the antecedent of "it" is in the various wordings, and what to do when the formal theorem itself is more self explanatory than would be the case in other situations.

The pull request as it stands mostly sticks with Norm's wording but with one tweak to deal with the issue of what kind of subsets are considered.

Your suggestion of letting other people review seems like a good one - perhaps they will have ideas about how to word this and/or whether this pull request is a good venue for getting into it.

Reply:
given that `e.` is like `<`, this says something like `A C_ B <-> A <_ B`
So a (perhaps uninituitive) comment could be: `` Equivalent versions of ` <_ ` for ordinals. ``
(I feel like such a comment would probably need some more explanation following it)

Otherwise I don't think the comment could really be shortened

Reply:
I like @benjub `s suggestion:  "A natural number is a subset of another natural number if and only if it belongs to its successor." It is not much longer than the current comment, but expresses the important fact that both directions hold.

Reply:
We have two votes for that wording, so that's good enough for me.